transports/webrtc-websys/src/connection.rs

```diff
@@ -179,18 +148,14 @@ impl StreamMuxer for Connection {
         mut self: Pin<&mut Self>,
         cx: &mut Context<'_>,
     ) -> Poll<Result<Self::Substream, Self::Error>> {
-        // Since this is NOT an initial Noise handshake outbound request (ie. Dialer)
-        // we need to create a new Data Channel WITHOUT negotiated flag set to true
-        // so use the Default DataChannelConfig
-        let config = DataChannelConfig::default();
-        self.inner.poll_create_data_channel(cx, config)
+        self.inner.poll_create_data_channel(cx)
     }
 
     fn poll_close(
         mut self: Pin<&mut Self>,
         _cx: &mut Context<'_>,
     ) -> Poll<Result<(), Self::Error>> {
-        debug!("connection::poll_close");
+        log::debug!("connection::poll_close");
```

Comment:
Nit: If you want to keep this, then I think it should be a trace.

```suggestion
        log::trace!("connection::poll_close");
```

Reply:
Got it, will do